### ISSUE TITLE
[@typescript-eslint/ban-ts-ignore] ban-ts-ignore not detecting /* @ts-ignore */ 

### ISSUE THREAD

**Issue by abilashs1990** [typescript-eslint/typescript-eslint#3513]
[@typescript-eslint/ban-ts-ignore] ban-ts-ignore not detecting /* @ts-ignore */ 

While `// @ts-ignore` is being detected and flagged as an error, `/* @ts-ignore */` is not being detected as an error and is passing the linter.

Rule that I have used:

```
{
    ...
    '@typescript-eslint/ban-ts-ignore': 2,
    ...
}
```

**Versions:** 
"@typescript-eslint/eslint-plugin": "^2.9.0",
"@typescript-eslint/parser": "^2.9.0",

**Screenshots of detection**
Test detects a ban on ts-ignore
![image](https://user-images.githubusercontent.com/42211409/121544587-79be9300-ca27-11eb-9f84-8ef4ea2905ed.png)

Test does not detect a ban on ts-ignore
![image](https://user-images.githubusercontent.com/42211409/121544743-935fda80-ca27-11eb-9404-324cba2094b9.png)

What am I doing wrong here?
